AdobeStockImage/etc/Api/AssetRepositoryInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Api;
+
+use Magento\AdobeStockImage\Api\Data\AssetInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Api\SearchResultsInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+interface AssetRepositoryInterface
+{
+    /**
+     * Save asset
+     * @api
```

Comment:
It's better to mark an interface as API and ensure it contains API methods only

Reply:
Hi @sivaschenko, I added this change. Can you check again please?

Reply:
can't see the change, could you check if you pushed everything

`@api` annotation should be added for interface and removed for methods

Reply:
Hi @sivaschenko, that's what I changed for this interface (http://prntscr.com/nvibl4) and what I see in my code https://github.com/igor-imaginemage/adobe-stock-integration/blob/feature/12-cleanup-stock-metadata/AdobeStockImage/Api/AssetRepositoryInterface.php

Can you confirm if this is correct?